api/v2/checluster_types.go

```diff
@@ -121,8 +121,17 @@ type CheClusterDevEnvironments struct {
 	// +kubebuilder:default:=-1
 	SecondsOfRunBeforeIdling *int32 `json:"secondsOfRunBeforeIdling,omitempty"`
 	// Disables the container build capabilities.
+	// DEPRECATED: Usage of the containerSecurityContext and podSecurityContext fields is recommended instead to configure privileges required for container builds.
```

Comment:
I am against marking this field as DEPRECATED. 
It is still a valid and easy way to enable container build capabilities.

Reply:
> It is still a valid and easy way to enable container build capabilities.

I agree @tolusha and am open to leaving it as-is.
@l0rd WDYT?

Perhaps its better to just modify the disableContainerBuildCapabilities documentation to mention that it overrides the containerSecurityContext (or have that noted on the containerSecurityContext documentation)?